modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notifications-for-default-platform-alerts_{context}"]
+= Configuring notifications for default platform alerts
+
+You can customize where and how Alertmanager sends notifications about default platform alerts.
+You can achieve this by editing the default configuration in the `alertmanager-main` secret in the `openshift-monitoring` namespace.
+
+[IMPORTANT]
+====
+The default configuration of the `alertmanager-main` secret does not generate notifications. Therefore, it is recommended to configure Alertmanager to receive notifications, as outlined in this section.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+
+.Procedure
+
+. Open the Alertmanager YAML configuration file:
+
+** To open the Alertmanager configuration from the CLI:
+
+.. Print the currently active Alertmanager configuration into file `alertmanager.yaml`:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring get secret alertmanager-main --template='{{ index .data "alertmanager.yaml" }}' | base64 --decode > alertmanager.yaml
+----
+
+.. Open the `alertmanager.yaml` file.
+
+** To open the Alertmanager configuration from the {product-title} web console:
+
+.. Go to the *Administration* -> *Cluster Settings* -> *Configuration* -> *Alertmanager* -> *YAML* page of the web console.
+
+. Edit the Alertmanager configuration:
++
+[source,yaml]
+----
+global:
+  resolve_timeout: 5m
+route:
+  group_wait: 30s #<1>
+  group_interval: 5m #<2>
+  repeat_interval: 12h #<3>
+  receiver: default
+  routes:
+  - matchers:
+    - "alertname=Watchdog"
+    repeat_interval: 2m
+    receiver: watchdog
+  - matchers:
+    - "service=<your_service>" #<4>
+    routes:
+    - matchers:
+      - <your_matching_rules> #<5>
+      receiver: <receiver> #<6>
+receivers:
+- name: default
+- name: watchdog
+- name: <receiver>
+  <receiver_configuration> #<7>
+----
+<1> Specify how long Alertmanager waits while collecting initial alerts for a group of alerts before sending a notification.
+<2> Specify how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
+<3> Specify the minimum amount of time that must pass before an alert notification is repeated.
+If you want a notification to repeat at each group interval, set the `repeat_interval` value to less than the `group_interval` value.
+However, the repeated notification can still be delayed, for example, when certain Alertmanager pods are restarted or rescheduled.
+<4> Specify the service that fires the alerts.
+<5> Specify labels to match your alerts.
+<6> Specify the receiver to use for the alerts.
+<7> Specify the receiver configuration.
++
+[IMPORTANT]
```

Comment:
Yes, but it would break the callouts when I would try to include it underneath callout 5 :disappointed:

Reply:
that is why I included it underneath the callouts

Reply:
You can try an open block in your next improvement PR if you have time to test them out. https://asciidoctor.org/docs/asciidoc-writers-guide/#open-blocks 
It's kind of pain to figure out exactly where you need them so that the list doesn't break (e.g., just around that callout, around the whole step, around all of the callouts, etc.), but once you get it working, it's a nice thing to have handy.